web/concrete/controllers/panel/detail/page/composer.php

```diff
@@ -128,7 +129,9 @@ protected function save() {
 
 			/// set the target
 			$configuredTarget = $pagetype->getPageTypePublishTargetObject();
-			$targetPageID = $configuredTarget->getPageTypePublishTargetConfiguredTargetParentPageID();
+			if ($configuredTarget instanceof PageTypePublishTargetConfiguration) {
+				$targetPageID = $configuredTarget->getPageTypePublishTargetConfiguredTargetParentPageID();
+			}
 			if (!$targetPageID) {
```

Comment:
> Or, $targetPageID = 0;

Yes, anything that would make `(!$targetPageID)` return `true`.
BTW, this won't change anything, since $targetPageID would be overwritten by the following `$targetPageID = $_POST['cParentID'];`